classes/Http/Form/SignupForm.php

```diff
@@ -88,22 +81,25 @@ public function validateSpeakerPhoto(): bool
             return true;
         }
 
+        /** @var HttpFoundation\File\UploadedFile $speakerPhoto */
+        $speakerPhoto = $this->taintedData['speaker_photo'];
```

Comment:
Introducing an explaining variable here so we can add a docblock (auto-completion).